calculator.rb

```diff
@@ -0,0 +1,142 @@
+def run_calculator
+  puts "Welcome to the calculator program!"
+  operation_select
+  run_again
+end
+
+def run_again
+  puts "\nWould you like to run the calculator again? (yes or no)"
+  input = gets.chomp
+    if input == "yes"
+      run_calculator
```

Comment:
Woah! Recursion! Neat!

One thing worth noting, when you finish the program the puts below will be hit as many times as you made calculations.

`Calculating...
5 * 4 = 20
Would you like to run the calculator again? (yes or no)
no
Cool, cya later. 
Cool, cya later. 
Cool, cya later. 
Cool, cya later. `